v03_pipeline/lib/misc/lookup.py

```diff
@@ -140,6 +140,7 @@ def remove_project(
             )
         ),
     )
+    ht = ht.filter(hl.any(ht.project_stats.map(hl.is_defined)))
```

Comment:
subtle difference between the two filters.... this one only checks whether a "project" is defined, the other goes deeper and checks for the presence of at least one family.